dart/lib/src/open_location_code.dart

```diff
@@ -118,13 +124,9 @@ bool isValid(String code) {
     return false;
   }
 
-  // Check code contains only valid characters.
-  for (var ch in code.codeUnits) {
-    if (ch > _decode.length || _decode[ch] < -1) {
-      return false;
-    }
-  }
-  return true;
+  var filterCallback = (ch) => !(ch > _decode.length || _decode[ch] < -1);
```

Comment:
Can you keep the implementation comment that says "Check code contains only valid characters."? It's not clear what this is doing on its own (not your fault, it was like that before).

Reply:
Sorry, removed it :c